provider/virtio_fs_utils.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import os
+
+from avocado.utils import process
+
+from virttest import data_dir
+from virttest import error_context
+from virttest import utils_misc
+
+LOG_JOB = logging.getLogger('avocado.test')
+
+
+def get_virtiofs_driver_letter(test, fs_target, session):
+    """
+    Get the virtiofs driver letter( Windows guest only )
+    :param test: QEMU test object
+    :param fs_target: virtio fs target
+    :param session: the session of guest
+    :return driver_letter: the driver letter of the virtiofs
+    """
+    error_context.context("Get driver letter of virtio fs target, "
+                          "the driver label is %s." % fs_target, LOG_JOB.info)
+    driver_letter = utils_misc.wait_for(
+        lambda: utils_misc.get_winutils_vol(session, fs_target),
+        timeout=120)
```

Comment:
```suggestion
    driver_letter = utils_misc.get_winutils_vol(session, label=fs_target):
```
Shall we need to wait for `get_get_win_disk_vol` twice?
I think the wait_for of `get_winutils_vol` is redandunt since the get_winutils_vol has did the same acition.
```
def get_winutils_vol(session, label="WIN_UTILS"):
 ...
    return wait_for(lambda: get_win_disk_vol(session,
                                             condition="VolumeName='%s'" % label), 240)
```